101-simple-windows-vm-monitoring-diagnostics/README.md

```diff
@@ -0,0 +1,21 @@
+# A Windows VM with the Azure diagnostics extension which enables monitoring and diagnostics capabilities 
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-simple-windows-vm-monitoring-diagnostics%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+
+Built by: [sbtron](https://github.com/sbtron)
```

Comment:
Deploy button is fine for anyone deploying from here. For other stuff below, the ACOM team will render the readme on Azure.com so a lot of this stuff ends up being doubled but they can exclude the deploy button from the readme.